bin/get_radius_secret

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
```

Comment:
Why do you need this file?  Can't you just do "echo SECRET"?

Reply:
as in the helper file itself?